### ISSUE TITLE
Is there any pretrained model on bdd100k? 

### ISSUE THREAD

**Issue by earcz** [ucbdrive/bdd100k#61]
Hi,

I am training YOLOv3 with different datasets for object detection in autonomous driving. bdd100k is what I need because of its classes (traffic light, traffic sign, car, person and so on). I don't want to train from scratch but transfer learning with YOLOv3. I am using _darknet53.conv74_ as pretrained model. However, it was trained on ImageNet. So I am looking for a model which was trained on **bdd100k in particular** or something similar dataset (for example kitti). Where can I find such kind of pretrained model? or is there any other solution to use different pretrained model to achieve good map results?

I am open to all recommendations to achieve a robust model with high accuracy. Any suggestion would be welcome to discuss.

**Comment by pytmar** [comment]
Hi ender, i was also looking for such thing like 4 months ago but i came with nothing so i try to find a model for self driving car only. This best architect i have found was [This](https://github.com/SurajDonthi/End-to-End-Model-for-Self-Driving-Cars) by nvidia. Your biggest challenge will be preparing data and fine tuning i wish you good luck. 
If you have no idea about where to start or how to start the things i have tried may help you [This](https://www.kaggle.com/umar47/complex-data-lab) i have done nothing worth you are  free to fork it. By the way if you are in turkey i am also here in istanbul connect me over my linkedin and we can be in touch about your results. Best of luck.

**Comment by earcz** [comment]
Hi Umar,

Nice to hear that you are striving to achieve something. 

I have organized data according to the folder structure of YOLOv3 for training. And today, just for testing to check if I organized data correctly, I trained YOLOv3 on bdd100k dataset for 400 epoch. Everything seems fine for now. I would like to discuss about these stuff with you. I will connect you on Linkedin tomorrow.

**Comment by wangzizhe** [comment]
Hi @earcz,

I'm facing the same problem. Have you finish training YOLOv3 on BDD100k, how's the result look like? Could you please share the .weights file if it is possible? 

Thanks a lot in advance!

**Comment by earcz** [comment]
> Have you finish training YOLOv3 on BDD100k

@wangzizhe Not really. I have trained it with multi-gpu up to 140.000 epochs. The model has reached %51 mAP. But I had to stop training for some reasons. That is to say, I did not complete the training. 


> Could you please share the .weights file if it is possible?

Yes, you can download it from [here](https://gofile.io/d/s2Ky7x).
